src/excel_adapter/excel_adapter.go

```diff
@@ -43,18 +46,30 @@ func getIP(sheet string, document internalModels.Document) string {
 	panic(fmt.Sprintf("excel adapter: getIP: Missing board %s IP\n", sheet))
 }
 
-func AddExpandedPackets(document internalModels.Document, objects ...models.FromBoards) {
-	globalInfo := models.GlobalInfo{BoardToIP: getBoardToIPs(document), UnitToOperations: make(map[string]string)}
+func Update(document internalModels.Document, objects ...models.FromDocument) {
+	globalInfo := getGlobalInfo(document)
+	for _, object := range objects {
+		object.AddGlobal(globalInfo)
+	}
 
 	for _, board := range getBoards(document) {
```

Comment:
Put in separate `addPackets` function

Reply:
Same as before, I don't like the idea of having one function for each for loop